### ISSUE TITLE
The applicability of W2D to the DenseNet model and whether it is suitable for pre-trained DenseNet models.

### ISSUE THREAD

**Issue by xmym778** [OoDBag/W2D#4]
Hello, I understand that W2D has shown good results with the ResNet model. My understanding is that in ResNet, the final feature maps represent high-level features, which W2D masks and updates through backpropagation. Is this mechanism suitable for DenseNet's densely connected architecture? Because DenseNet's output feature maps contain information from lower layers as well. Also, when using W2D with pre-trained models, would increasing the number of epochs be necessary to ensure sufficient adjustment? I may have some misunderstandings, apologies for that.

**Comment by xmym778** [comment]
使用的是翻译，可能不是很精确，hahaha
